src/test/java/com/cegeka/academy/repository/InvitationRepositoryTest.java

```diff
@@ -17,17 +19,33 @@
 public class InvitationRepositoryTest {
 
     @Autowired InvitationRepository invitationRepository;
+    @Autowired EventRepository eventRepository;
+    @Autowired UserRepository userRepository;
 
     @Test
     public void testAddInvitation(){
         Invitation invitation = new Invitation();
```

Comment:
Pentru a modulariza clasa de test ai putea face cate o metoda care sa iti returneze o noua instatnta pentru fiecare entitate(_Invitation_, _Event_,  _User_). Eventual poti face o clasa de test utilitara care sa iti ofere obiecte dummy si ar putea fi folosita si in alte clase de test.